public/app/features/alerting/unified/api/alertmanagerApi.ts

```diff
@@ -274,11 +274,13 @@ export const alertmanagerApi = alertingApi.injectEndpoints({
           }),
         }));
       },
+      providesTags: ['ContactPointsStatus'],
     }),
     // Grafana Managed Alertmanager only
     // TODO: Remove as part of migration to k8s API for receivers
     getContactPointsList: build.query<GrafanaManagedContactPoint[], void>({
       query: () => ({ url: '/api/v1/notifications/receivers' }),
+      providesTags: ['ContactPoint'],
```

Comment:
We should probably use or add `AlertmanagerConfiguration` here since the `updateAlertmanagerConfiguration` endpoint might also affect this.

Reply:
:+1: I think it should also?/instead? be done in the other place. Mutating the alertmanager configuration should invalidate the tags of every other endpoint that matters.
This endpoint doesn't "provide" alertmanager configs, but mutating the AM config should invalidate any entities you have relating to contact points, time intervals etc.

WDYT?